README.md

```diff
@@ -34,9 +34,25 @@ make debug
 We do not provide packages for install at this time. Contact us if you're
 interested in those.
 
-### Run
+## Run
 
-You can easily run the program with sample config provided in `config/` directory by running `./run.sh`.
+Run the full stack `carbonapi` -> `zipper` -> `go-carbon` with:
```

Comment:
in this setup you don't need a zipper.
carbonapi -> go-carbon should work fine. Zipper needed if you have more than 1 go-carbon instance.

Reply:
No, we should have our setup as the default, and not the simplest stack missing a key component.

Reply:
It makes sense to have a simplified setup as a special case.

But in general case, we need `zipper` to have a full stack for testing changes and running system tests.